contracts/libraries/ShareMath.sol

```diff
@@ -56,19 +48,14 @@ library ShareMath {
         Vault.DepositReceipt memory depositReceipt,
         uint256 currentRound,
         uint256 pps,
-        uint8 decimals
-    ) internal pure returns (uint128 unredeemedShares) {
+        uint256 decimals
+    ) internal pure returns (uint256 unredeemedShares) {
         if (depositReceipt.round > 0 && depositReceipt.round < currentRound) {
             uint256 sharesFromRound =
                 underlyingToShares(depositReceipt.amount, pps, decimals);
 
-            assertUint104(sharesFromRound);
-
-            uint256 unredeemedShares256 =
+            return
                 uint256(depositReceipt.unredeemedShares).add(sharesFromRound);
-            assertUint128(unredeemedShares256);
-
-            unredeemedShares = uint128(unredeemedShares256);
         } else {
```

Comment:
can just do` return unredeemedShares = depositReceipt.unredeemedShares;` instead of else